plugins/inputs/cpu/cpu.go

```diff
@@ -37,8 +41,13 @@ func (c *CPUStats) Gather(acc telegraf.Accumulator) error {
 	now := time.Now()
 
 	for _, cts := range times {
-		tags := map[string]string{
-			"cpu": cts.CPU,
+		tags := make(map[string]string)
+		tags["cpu"] = cts.CPU
```

Comment:
Please keep this part as it was before, you can do the logic below (if core-tags avail etc) anyway.